### ISSUE TITLE
⌨ Unclear tab focus in processes

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#14541]
### ⌨ Accessibility and Expected Behaviour

🛡️Fable tester reported: On some of the flow, it was reasonable to see (based on the colour contrast) but in this step, (from processes navigate to screening and assessment) it is near impossible to see the focus. It is not well contrasted, in fact is a dark colour against the dark background, and it is not bold or evident that focus is visible. Without the proper focus and visibility I am forced to use my mouse/trackpad which is more effort than there should be. 

After further investigation with @esizer , we believe this refers to both the tab focus and the menu focus for mobile/zoom view. 

### WCAG 2.1 Criteria

2.4.7 Focus Visible

### 📋 Steps to Reproduce

1. sign in
2. change view to dark mode
3. switch to admin or community view
4. zoom in to 200% 
5. open a process 
6. navigate to Screening and assessment by tabbing 
7. switch focus to menu by tabbing

### What operating systems are you encountering the problem on?

macOS

### What browsers are you encountering the problem on?

Safari

### What assistive technologies are you encountering the problem on?

OS Magnification

### 📸 Screenshot

<img width="1582" height="785" alt="Image" src="https://github.com/user-attachments/assets/7d78bd22-c859-4344-b531-4a107e4c251d" />
<img width="1280" height="615" alt="Image" src="https://github.com/user-attachments/assets/10453cdb-17f2-4ea4-9e01-a9ada73074b9" />

### 🙋‍♀️ Proposed Solution

The focus area needs to be more visible, maybe the entire tib turns to the focus colour? 

The hamburger menu needs an outline or a distinguished colour from the background.

**Comment by BrookeCameron** [comment]
@Jerryescandon can you take a look at the tab focus?

**Comment by gobyrne** [comment]
It's not so clear to me how the mobile menu is involved here. Those tabs could have more focus indication though. Should the text appear highlighted as well?

**Comment by BrookeCameron** [comment]
the menu is hard to see with the same colour background as the button

**Comment by BrookeCameron** [comment]
@JoshBeveridge will be addressing the tab focus state to span the entire tab (not just the top 8th) in the component redesign.
